src/arrayMethodSort.js

```diff
@@ -1,12 +1,26 @@
-'use strict';
+"use strict";
 
 /**
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) => String(a) > String(b)
```

Comment:
Move this expression `(a, b) => String(a) > String(b)` to helper function with meaningful function naming